Updated IBM Resilient integration (#27004)
demisto/content: Packs/IBMResilientSystems/ReleaseNotes/1_1_8.md

#### Integrations
##### IBM Resilient Systems
- Updated the Docker image to: *demisto/resilient:2.0.0.62392*.

> - Added a parameter max_fetch for the integration (default value=50) to control the maximum amount of incident to fetch in one go.

Reviewer: ```suggestion
- Added the *max_fetch* parameter (default value=50) to control the maximum amount of incidents to fetch at a time.
```
